### DISCUSSION TITLE
consult-grep family integrates badly with literally-visited files

### DISCUSSION THREAD

**Discussion by hiecaq** [minad/consult#1005] [Q&A] [answered]
step to test:
1. Visit some file literally with `find-file-literally`
2. C-u consult-grep with this file as input and search for some words in the file
3. When the candidate is picked to perform the jump, A prompt shows up asking "Do you want to revisit the file normally now"
4. Answering "yes" reverts and revisits the file normally, answering "no" aborts the jump.

I think the cause is https://github.com/minad/consult/blob/d8888bb67f881a3c4855c9ce7224de18a7dc3901/consult.el#L4751 where `find-file-noselect` is called with the RAWFILE argument always default to `nil`.

The reason I wish this is fixed is because, visiting a large file (>hundreds of MBs) literally increases the browsing speed significantly, and using `consult-ripgrep` on this file to jump around the lines is also much faster than `consult-line` on my machine. It'd be very nice if they can work together.

**Comment by minad** [comment, marked as answer]
Makes sense. I quickly tried your suggestion in https://github.com/minad/consult/commit/bc11084deface58ab9241a2111e9579c948da33f. I would appreciate if you check if it works as expected now.

**Comment by hiecaq** [comment]
I tested it with the latest commit today, and it works great. Thanks!
